salt/modules/puppet.py

```diff
@@ -166,7 +156,7 @@ def run(*args, **kwargs):
 
     # new args tuple to filter out agent/apply for _Puppet.arguments()
     buildargs = ()
-    for arg in range(len(args)):
+    for arg in range(len(args)):  # pylint: disable=C0200
```

Comment:
yeah, this instance doesn't even need enumerate because we're just using the value from args anyway. I'll look at all of these and resolve them though.